src/docs/product/data-management-settings/dynamic-sampling/dsla/index.mdx

```diff
@@ -0,0 +1,35 @@
+
+Sentry's **Dynamic Sampling** feature allows you to view more accurate performance metrics based on all the transaction events you send to Sentry (called _processed events_), while retaining a smaller number of more relevant transaction events that you can explore in full detail (called _indexed events_).
+
+With dynamic sampling, you control which transactions Sentry retains by setting _server-side sampling_ rules and rates, so you see more of the transactions you want to explore further, and less of the ones you don’t. In addition, dynamic sampling allows you to manage your sampling configuration centrally, server-side, and lets you make changes without the need to reconfigure the SDK and redeploy your application.
+
+![The Dynamic Sampling page with a list of rules](dynamic-sampling-dsla.png)
+
+> _[A trace](/product/sentry-basics/tracing/distributed-tracing/) represents the record of an entire operation you want to measure or track, like a page load, an instance of a user completing some action in your application, or a cron job in your backend. A trace consists of one or more transactions and the trace can be distributed across multiple services or projects._
+
+Dynamic sampling allows you to retain a smaller number of more relevant transactions that you can explore in full detail, without giving up the accuracy of performance metrics that are based on larger set of all the transactions you send to Sentry.
+
+Because every project is different, you can set multiple sample rules with different sample rates per project. For example, you may need more transactions from high converting pages, critical API endpoints, or you may need to focus on latency issues from your latest release.
+
+Dynamic sampling also makes your SDK (client-side) and your server-side sampling configuration largely independent of each other. This helps if you're building native applications with many active releases at a given time.
+
+## Traces & Propagation of Sampling Decisions
+
+When Sentry makes sampling decisions, we do so for entire traces. Instead of looking at a particular transaction, we look at the entire group of transactions in a trace. Sampling transactions using transaction traces is ideal because it allows you to understand transactions in the context of your whole system.
+
+Since we sample traces, sampling rules are based on the trace context. This means that sampling decisions are based on the information extracted from the _system that initiated the transaction_. The head transaction in a trace determines the trace context for all following transactions in that trace. No information can be changed, added, or deleted after the first propagation. The trace context is bound to only one particular trace, and all the transactions that are part of this trace.
+
+## Next Steps
```

Comment:
Are we planning to add more steps in the future? otherwise I would remove it or call it `Next Step`

Reply:
This is the hidden version of the page with the old content, so that <PageGrid /> component creates the list of pages under it. But on the updated version of the page [here](https://github.com/getsentry/sentry-docs/pull/5688/files#diff-b0d530f14be8d4f60b5b1568816e62f96e31ba257e1a68f725bb669be8b00925), I've updated that spot to get rid of the Next Steps heading and replaced it with a Getting Started heading.